ToolBox/Program.cs

```diff
@@ -261,7 +261,8 @@ public static void Main(string[] args)
                         SmartInsiderProgram.SmartInsiderConverter(
                             DateTime.ParseExact(GetParameterOrExit(optionsObject, "date"), "yyyyMMdd", CultureInfo.InvariantCulture),
                             GetParameterOrExit(optionsObject, "source-dir"),
-                            GetParameterOrExit(optionsObject, "destination-dir"));
+                            GetParameterOrExit(optionsObject, "destination-dir"),
+                            GetParameterOrDefault(optionsObject, "source-meta-dir", Path.Combine(Globals.DataFolder, "alternative", "smartinsider")));
```

Comment:
Maybe could send in `null` here, to avoid default value duplication with the implementation at `SmartInsiderConverter`?

Reply:
Good idea 👍